### PR TITLE
node missing type def under perf_hooks.d.ts for the RH interface. 

### PR DESCRIPTION
**Pull request by meeklowebb**
✅ Use a meaningful title for the pull request. Include the name of the package modified.
✅ Test the change in your own code. (Compile and run.)
✅ Add or edit tests reflect the change.
✅ Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
✅ [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
✅ Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
✅ If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

According to NodeJS documentation https://nodejs.org/api/perf_hooks.html
The function should exist under corresponding interface

image
image

### CODE REVIEW

**Comment by meeklowebb** [comment]
???

image

**Comment by yoursunny** [comment]
> ??? node-sass

Wait for #60974.

**Comment by meeklowebb** [comment]
> 
Ok thank you !

File: types/node/test/perf_hooks.ts [hunk 1]
```diff
@@ -100,11 +100,7 @@ histogram.record(123);
 histogram.record(123n);
 histogram.recordDelta();
 
-/**
- * Now intelligence is working for the "add" function added
- * under interface RecordableHistogram
- * under perf_hooks.ts
- */
+//intelligence is working
```
Comment (peterblazejewicz) [inline types/node/test/perf_hooks.ts]:
```suggestion
// intelligence is working
```
more readable, current one, infamously, is a style (missing space) that MS Visual Studio is using as a default (but not VSCode IMO)

Comment (meeklowebb) [reply types/node/test/perf_hooks.ts:103]: Ok, thank you!

**Review by mcollina — APPROVED** [review]
lgtm

**Review by peterblazejewicz — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]

**Comment by meeklowebb** [comment]
Ready to merge

**Pull request merged**